lib/dcell/utils.rb

```diff
@@ -11,6 +11,10 @@ def full_const_get(name)
         end
         obj
       end
+
+      def uuid
+        SecureRandom.uuid
```

Comment:
Just use Celluloid.uuid. It's fast ;)

It uses a bit of random data grabbed at start and a thread-local counter. If you'd like for the UUIDs to be indistinguishable you can hash them.

Benchmarks:

https://gist.github.com/tarcieri/2190462